source/Octopus.Tentacle.Tests.Integration/CapabilitiesServiceV2Test.cs

```diff
@@ -8,13 +8,17 @@ namespace Octopus.Tentacle.Tests.Integration
 {
     public class CapabilitiesServiceV2Test : IntegrationTest
     {
-        [TestCase(null)] // The version of tentacle compiled from the current code.
-        [TestCase("5.0.4")] // First linux Release 9/9/2019
-        [TestCase("5.0.12")] // The autofac service was in octopus shared.
-        [TestCase("6.3.451")] // the autofac service is in tentacle, but tentacle does not have the capabilities service.
-        public async Task CapabilitiesFromAnOlderTentacleWhichHasNoCapabilitiesService_WorksWithTheBackwardsCompatabilityDecorator(string version)
+        [TestCase(TentacleType.Polling, null)] // The version of tentacle compiled from the current code.
+        [TestCase(TentacleType.Polling, "5.0.4")] // First linux Release 9/9/2019
+        [TestCase(TentacleType.Polling, "5.0.12")] // The autofac service was in octopus shared.
+        [TestCase(TentacleType.Polling, "6.3.451")] // the autofac service is in tentacle, but tentacle does not have the capabilities service.
+        [TestCase(TentacleType.Listening, null)]
+        [TestCase(TentacleType.Listening, "5.0.4")]
+        [TestCase(TentacleType.Listening, "5.0.12")]
+        [TestCase(TentacleType.Listening, "6.3.451")]
+        public async Task CapabilitiesFromAnOlderTentacleWhichHasNoCapabilitiesService_WorksWithTheBackwardsCompatabilityDecorator(TentacleType tentacleType, string version)
```

Comment:
I updated this to run against listening and polling as well
It uses a very specific set of versions, so left it as TestCase for that reason